### PR TITLE
Mobula integration

### PR DESCRIPTION
**Pull request by tdroxler**
resolves https://github.com/alephium/explorer-backend/issues/557

Thx to Mobula, we can now find the price for almost all our token list, only ~40 tokens don't have a price.

In this PR we are now taking the token list and grab the price for all of them from mobula.

As previously we cache the data and reload every 5min.

### CODE REVIEW

File: app/src/main/scala/org/alephium/explorer/service/MarketService.scala [hunk 1]
```diff
@@ -30,46 +30,50 @@ import org.alephium.explorer.cache._
...
-import org.alephium.util.{Duration, Math, Service, TimeStamp}
+import org.alephium.protocol.Hash
+import org.alephium.protocol.model.{Address, ContractId}
+import org.alephium.util.{discard, Duration, Hex, Math, Service, TimeStamp}
 
 trait MarketService {
   def getPrices(
       ids: ArraySeq[String],
-      currency: String
+      chartIurrency: String
```
Comment (h0ngcha0) [inline app/src/main/scala/org/alephium/explorer/service/MarketService.scala:40]: [Nit] `chartCurrency` ?

File: app/src/main/scala/org/alephium/explorer/service/MarketService.scala [hunk 2]
```diff
@@ -102,7 +106,7 @@ object MarketService extends StrictLogging {
      * We use our `AsyncReloadCache` that always return the latest cached value
      * even if it's expired, like this we guarantee to always return fast a data.
      * We use an `Either` value because we cannot control what's returned by
-     * coingecko and it might be that their endoints return something else.
+     * coingecko and it might be that their endoints ret60rn something else.
```
Comment (h0ngcha0) [inline app/src/main/scala/org/alephium/explorer/service/MarketService.scala:109]: [Nit] Is this intentional?

**Review by h0ngcha0 — COMMENTED** [review]
LGTM, only minor comments.

File: app/src/main/scala/org/alephium/explorer/service/MarketService.scala (hunk 2 above)
Comment (tdroxler) [reply app/src/main/scala/org/alephium/explorer/service/MarketService.scala:109]: oops definitively not

**Pull request merged**